gmusicapi/test/server_tests.py

```diff
@@ -964,8 +961,8 @@ def mc_search_store_no_playlists(self):
 
         with Check() as check:
             for type_, hits in res.items():
-                if ((not test_subscription_features() and
-                     type_ in ('artist_hits', 'song_hits', 'album_hits'))):
+                if (not test_subscription_features() and
+                        type_ in ('artist_hits', 'song_hits', 'album_hits')):
```

Comment:
I don't mind the change, but i'm curious: did pyupgrade flag this? I read about the extra paren trick somewhere long ago as a way of avoiding the condition/body indentation warning.

Reply:
Yeah, `pyupgrade` noticed the extra parenthesis. I had to add the extra indentation to get past `flake8` on Travis.

I'd adopt [Black](https://github.com/psf/black) for formatting everything instead of dealing with this. However, that's more a thing for you to decide and do than for me to send a PR for, as it would be impossible to review that I didn't do anything else also, and hid it in the noise of reformatting everything.

Reply:
Yeah, I've seen a fair amount of adoption of black lately. I hadn't really considered it for my projects but I suppose it could make contributing easier.